tests/core/selection/manual/optimizationlist.html

```diff
@@ -0,0 +1,39 @@
+<h2>Iframe editor</h2>
+<div id="editor1">
+	<ol>
+		<li>1</li>
+		<li>2
+		<ol>
+			<li>1</li>
+		</ol>
+		</li>
+		<li>&nbsp;</li>
+	</ol>
+
```

Comment:
Let's remove this unnecessary empty line. This comment applies also to the second list below. 
```suggestion
```